lib/transactions/signed/SignedTransaction.d.ts

```diff
@@ -0,0 +1,39 @@
+import { Transactions } from '../Transactions';
+export declare abstract class SignedTransaction extends Transactions {
+    static prepareResponse(result: {
+        transaction_id: string;
+        processed: {
+            block_num: number;
+            action_traces: Array<{
+                receipt: {
+                    response: string;
+                };
+            }>;
+        };
+    } | any, includeTrxId?: boolean): any;
+    static parseProcessedResult(processed: {
+        action_traces: Array<{
+            receipt: {
+                response: string;
+            };
+        }>;
+    }): any;
+    abstract ENDPOINT: string;
+    abstract ACTION: string;
+    abstract ACCOUNT: string;
+    abstract getData(): any;
+    execute(privateKey: string, publicKey: string, dryRun?: boolean): Promise<any>;
```

Comment:
And `expirationOffset = Constants.defaultExpirationOffset` parameter on execute method